src/coreclr-debug.ts

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as child_process from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+var _coreClrDebugDir: string;
+var _debugAdapterDir: string;
+var _channel: vscode.OutputChannel;
```

Comment:
Best to use 'let' rather than 'var'. I'm working on moving things to ES6. Note that there's a tslint gulp task included that you can run in VS Code to find minor issues like this.